components/rmrk/Gallery/GalleryItemRelated.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div>
+    <div class="my-5" v-if="nftRelated.length >= MIN_NFTS">
+      <p class="subtitle is-size-4">More From This Collection</p>
+      <CarouselCardList :nfts="nftRelated" />
+    </div>
+    <div class="my-5" v-if="nftVisited.length >= MIN_NFTS">
+      <p class="subtitle is-size-4">Visited NFTs</p>
+      <CarouselCardList :nfts="nftVisited" />
+    </div>
```

Comment:
🙈

Reply:
This should be component
```vue
    <div class="my-5" v-if="nftRelated.length >= MIN_NFTS">
      <p class="subtitle is-size-4">More From This Collection</p>
      <CarouselCardList :nfts="nftRelated" />
    </div>
```

also missing translations

Reply:
@vikiival noted. do you have any concerns in the `async fetch()` section?